string action
# id of object (max 255)
int16 object
---
bool   success
string response
